### PR TITLE
add wgCreatePageUwUseVE to ManageWikiSettings

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

**Comment by Universal-Omega** [comment]
Needs added to LocalSettings.php also.

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -848,6 +848,15 @@
 		'help' => 'Whether to expand templates of the preloaded text.',
 		'requires' => [],
 	],
+	'wgCreatePageUwUseVE' => [
+		'name' => 'CreatePageUw Use VisualEditor',
+		'from' => 'createpageuw',
+		'type' => 'check',
+		'overridedefault' => 'false',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:855]:
```suggestion
		'overridedefault' => false,
```

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -848,6 +848,15 @@
...
+		'section' => 'editing',
+		'help' => 'If true, user will be redirected to VisualEditor.',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:857]:
```suggestion
		'help' => 'If enabled, user will be redirected to VisualEditor.',
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by ugochimobi** [comment]
> Needs added to LocalSettings.php also.

Done

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**